README.md

```diff
@@ -22,7 +22,19 @@ On Windows 10 you can either build for win32 or use XMing and run your builds in
 
 For more information about how to build on the various systems, refer to the platform specific docs in the `docs` folder!
 
-# Overview
+# More docs!
+
+Refer to the OS specific docs:
+
+* [Linux users](docs/Linux.md)
+* [Windows users](docs/Windows.md) or if you are [Visual Studio user](docs/VisualStudio.md)
+* [MacOSX users](docs/MaxOSX.md)
```

Comment:
A bit late, but there's a typo here: "MaxOSX". (Also I think it's called "macOS" now...)